src/lib-components/overlays/Tooltip.vue

```diff
@@ -36,9 +37,9 @@ const hoverPopover = (e: MouseEvent, open: boolean): void => {
 
 <template>
   <Popover :position="position" :as="as">
-    <template #button="{ open, close }">
+    <template #button="{ open, close }: { open: boolean, close: () => void }">
       <div
-        class="leading-none w-4 h-4"
+        class="leading-none relative w-4 h-4"
```

Comment:
The "large target" hack I added to the tooltip was ill placed under some scenarios, this should keep it well placed on top of the icon.